### PR TITLE
Instructions to run locally for development

### PR DESCRIPTION
**Pull request by leahnp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,17 @@ Message me with any comments or suggestions!
 Info on FluentBit: https://github.com/fluent/fluent-bit
 
 Info on Golang Output Plugins for FluentBit: https://github.com/fluent/fluent-bit-go
+
+Run locally for development:
```
Comment (yenicapotediaz) [inline README.md]: Maybe add ### to make it bigger

File: README.md [hunk 2]
```diff
@@ -8,3 +8,17 @@ Message me with any comments or suggestions!
...
+
+Follow the instructions [here](https://kafka.apache.org/quickstart) to start zookeeper and kafka locally on your computer.
+
+Download [Fluent-bit](http://fluentbit.io/download/) locally. [Build Fluent-bit](http://fluentbit.io/documentation/0.12/installation/build_install.html). Test Fluent-bit is installed and built properly by running: `bin/fluent-bit -i random -o stdout` from your Fluent-bit build directory. This command starts Fluent-bit, tells it to use "random" for the input plugin and "stdout" for the output plugin.
+
+Now we want to run Fluent-bit with the kafka output plugin. `bin/fluent-bit -v -e /path to.../fluent-bit-kafka-output-plugin/out_kafka.so -i random -o out_kafka`. Remember to change the broker list in the plugin to: "localhost:9092" and re-"MAKE".
```
Comment (yenicapotediaz) [inline README.md]: the `/path to.../fluent-bit-kafka-output-plugin/out_kafka.so` seems to be 2 different parts.
I'd recommend changing it to `<path to ../fluent-bit-kafka-output-plugin/out_kafka.so>`

**Comment by leahnp** [comment]
@yenicapotediaz thanks! Fixed the issues.

**Comment by yenicapotediaz** [comment]
@guineveresaenger looked over my shoulder and approved it as well :)

**Pull request merged**